app/scripts/views/mixins/signin-mixin.js

```diff
@@ -130,6 +130,7 @@ define(function (require, exports, module) {
         this.relier.set('uid', account.get('uid'));
       }
 
+      this.logEvent('signin.success');
```

Comment:
@shane-tomlinson this is not exactly what we talked about earlier, but if I replace         `this.logViewEvent('success');` with `this.logEvent('signin.success');` the oauth tests [here](https://github.com/mozilla/fxa-content-server/blob/9e8e7f079fed29f8bceb6e4bf61fd6c231132e81/app/tests/spec/views/oauth_sign_in.js#L149) fail because `oauth.signin.success` metric is not emitted. Will logging the metric in the way above cause duplicates? The same issue holds for signin and signup.